src/tools/schema2kotlin.ts

```diff
@@ -304,6 +306,6 @@ ${typeDecls.length ? typeDecls.join('\n') + '\n' : ''}`;
   }
 
   private getType(type: string): string {
-    return this.opts.wasm ? `Wasm${type}` : `Jvm${type}`;
+    return this.opts.wasm ? `Wasm${type}` : `${type}`;
```

Comment:
I know there's been shuffling around Jvm, I don't have enough of the latest background to validate what's being done here, @csilvestrini to comment.

Reply:
Jvm* types were added because of WASM. I hoped the the serialize/deserialize entitySpec stuff up to EntitySpec/Entity. 

The point here is that we don't want core to depend on SDK. These Handle classes are in Core, and need to invoke serialize/deserialize, ergo, they need to be in core. Which is now safe to do because of Cameron's changes.